src/services/info/location-info.ts

```diff
@@ -0,0 +1,59 @@
+import { urlStorage } from '@/services/state-persistor/storage/url-storage'
+import { transform } from 'ol/proj'
+import { Coordinate } from 'ol/coordinate'
+import { Projection } from 'ol/proj'
+import { containsCoordinate } from 'ol/extent'
+
+import { getElevation } from '@/components/draw/feature-measurements-helper'
+
+export const INFO_PROJECTIONS = {
+  'EPSG:2169': 'Luref',
+  'EPSG:4326': 'Lon/Lat WGS84',
+  'EPSG:4326:DMS': 'Lon/Lat WGS84 DMS',
+  'EPSG:4326:DMm': 'Lon/Lat WGS84 DM',
+  'EPSG:3263*': 'WGS84 UTM',
+}
+
+export const LIDAR_EXTENT = [46602, 53725, 106944, 141219]
+export const LIDAR_EXTENT_SRS = 'EPSG:2169'
+
+export async function queryInfos(location: Coordinate, fromCrs: Projection) {
+  const clickCoordinateLuref = transform(location, fromCrs, 'EPSG:2169')
```

Comment:
use constant

Reply:
I don't see much benefit, since the value of the constant will never change (projections are standard strings) but why not